invenio_communities/assets/semantic-ui/js/invenio_communities/members/components/SearchFilters.js

```diff
@@ -0,0 +1,64 @@
+/*
+ * This file is part of Invenio.
+ * Copyright (C) 2022 CERN.
+ *
+ * Invenio is free software; you can redistribute it and/or modify it
+ * under the terms of the MIT License; see LICENSE file for more details.
+ */
+
+import { DropdownFilter } from "@js/invenio_communities/members/components/SearchDropdowns";
+import PropTypes from "prop-types";
+import React, { Component } from "react";
+import { withState, Sort } from "react-searchkit";
+
+export class SearchFilters extends Component {
+  render() {
+    const {
+      sortOptions,
+      currentQueryState,
+      updateQueryState,
+      currentResultsState,
+      customCmp,
+      customFilters,
+    } = this.props;
+    const filters = customFilters
+      ? customFilters
+      : currentResultsState.data.aggregations;
+
+    return (
+      <>
+        {Object.entries(filters).map((filter) => (
+          <DropdownFilter
+            key={filter[0]}
+            filterKey={filter[0]}
+            filterLabel={filter[1].label}
+            filterValues={filter[1].buckets}
+            currentQueryState={currentQueryState}
+            updateQueryState={updateQueryState}
+            loading={currentResultsState.loading}
+            size="large"
+            className="fluid-mobile"
+          />
+        ))}
+
+        {sortOptions && <Sort values={sortOptions} />}
```

Comment:
I think this and the custom component can be moved in the layout component, wdyt?